bitrise.yml

```diff
@@ -179,7 +179,7 @@ workflows:
             "-parallel-testing-enabled" "NO" "-parallel-testing-worker-count" "2"
             "COMPILER_INDEX_STORE_ENABLE=NO" "CODE_SIGN_IDENTITY=" 
             "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO"'
-        - xctestrun: "Users/vagrant/git/DerivedData/Build/Products/Fennec_Smoketest2_iphonesimulator17.0-arm64.xctestrun"
+        - xctestrun: "Users/vagrant/git/DerivedData/Build/Products/Fennec_Smoketest2_iphonesimulator17.0-arm64-x86_64.xctestrun"
```

Comment:
I'm not sure about this change... for the Unit test it is Fennec_UnitTest_iphonesimulator17.0-arm64.xctestrun... it has always been the same for all except fo the test plan name part...